hack/externalnode/install-vm.ps1

```diff
@@ -174,34 +192,72 @@ function UpdateAgentConf() {
     }
 }
 
+function ClearOVSConfig() {
+    Log "Deleting OVS bridge $OVSBridge"
+    $AntreaSwitch = "antrea-switch"
+    Stop-Service $AntreaAgent
+    try {
+        $adapterName = (Get-VMNetworkAdapter -SwitchName $AntreaSwitch -ManagementOS).Name
+        ovs-vsctl.exe del-br $OVSBridge
+    }  catch {
+        Log "Failed to get VMSwitch $AntreaSwitch , rc $_"
+        exit 1
+    }
+
+    try {
+        Remove-VMSwitch -ComputerName $(hostname.exe) $AntreaSwitch  -Force
+    } catch {
+        Log "Ignore error while removing VMSwitch, rc $_"
+    }
+
+    try {
+        Rename-NetAdapter -Name "$adapterName~" -NewName "$adapterName"
+    }  catch {
+        Log "Failed to rename network adapter $adapterName~ to $adapterName, rc $_"
+        exit 1
+    }
+    Start-Service $AntreaAgent
```

Comment:
The Cleanup routine will Stop antrea-agent service, performs cleanup, and then Start antrea-agent service.
We cannot gurantee any order, that is the reason, cleanup routine explicitly calls stop service so that antrea-agent does not start by itself.